aiida_vasp/parsers/tests/test_parsevasp.py

```diff
@@ -0,0 +1,84 @@
+"""Unittests for ParseVasp."""
```

Comment:
can you rename this file `test_default_parser.py`, to make it clear that this is the default parser being tesed?
Also, maybe change the docstring to "Unit tests for the default `VaspParser`, using ParseVasp".

Reply:
Actually, I think this file should be merged with `test_vasp_parser.py`, since both are testing the default parser. The name should stay `test_vasp_parser.py` In this case

Reply:
There seems to be currently no test for the alternatives system.
Specifically testing that alternatives will get parsed instead and still appended to the correct output node link.

This can also be split out of this PR in a separate issue.

Reply:
Done, tests will trickle in now that it actually works.